fork-client/build.gradle

```diff
@@ -0,0 +1,10 @@
+apply plugin: 'java-library'
+apply from: '../build-scripts/gradle-mvn-push.gradle'
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+}
+
+//noinspection GroovyUnusedAssignment
+sourceCompatibility = rootProject.sourceCompatibility
```

Comment:
We don't need to use Java 8, since this is just an annotation. We can keep 7 as the lower bound of source compatibility.

Reply:
👍